deploy/kubernetes/images/prow-gke-release-staging-rc-1-19/image.yaml

```diff
@@ -31,8 +31,8 @@ metadata:
   name: imagetag-csi-gce-driver-prow-rc
 imageTag:
   name: gke.gcr.io/gcp-compute-persistent-disk-csi-driver
-  newName: gcr.io/gke-release-staging/gcp-compute-persistent-disk-csi-driver
-  newTag: "v1.2.1-rc1"
+  newName: k8s.gcr.io/cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver
+  newTag: "v1.2.2"
```

Comment:
These versions match what we're shipping with the managed driver in GKE, so it's probably good to keep them.

The manifests in k/k are way out of date. The intent is to test that k/k hasn't broken the CSI interface rather than testing latest versions of PD CSI IIUC.

It's not clear to me that the work on putting the PD CSI driver in cloud-provider-gcp is going to mean we should remove the testing manifests from k/k or not?

Reply:
yes as part of that effort we'll remove the manifests from k/k